Src/AutoFixtureUnitTest/AbstractRecursionIssue/Repro.cs

```diff
@@ -65,7 +65,7 @@ public void Workaround()
                     typeof(ItemBase),
                     typeof(FunkyItem)));
 
-            Assert.DoesNotThrow(() => fixture.Create<FunkyItem>());
+            AssertEx.DoesNotThrow(() => fixture.Create<FunkyItem>());
```

Comment:
Have we tested whether `AssertEx` works correctly? Are we sure we're not running into [False Positives](http://xunitpatterns.com/false%20positive.html) or [False Negatives](http://xunitpatterns.com/false%20negative.html)?